web_drop_target/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+===================
+Drop target support
+===================
+
+This module was written to extend the functionality of the web client to support dropping local files into the web client.
```

Comment:
+This module extends the functionality of the web client to support dropping local files into the web client.

Reply:
You can take the occasion of using new README fragments system